### PR TITLE
Add dialect Cearences

### PR DESCRIPTION
**Pull request by Arieviloo**
## Pre-Submission Checklist

- You have only one commit (if not, squash them into one commit).
- All new and existing tests pass the command `yarn test` and `yarn lint`.

### CODE REVIEW

File: dialects/cearences.json [hunk 1]
```diff
@@ -1,72 +1,113 @@
 [
-	{
```
Comment (mvfsillva) [inline dialects/cearences.json]: Why was there a replace here?

File: dialects/cearences.json [hunk 2]
```diff
@@ -1,72 +1,113 @@
...
-		"dialect": "Malamanhado",
-		"meanings": ["Desleixado","Desajeitado"],
-		"examples": ["Macho, tu só vem pra aula malamanhado, venha mais arrumadinho, rapaz."]
-  },
-  {
-		"dialect": "Rebolar no mato",
-		"meanings": ["Jogar fora"],
-		"examples": ["Querida, esse queijo já passou o prazo de validade, tá bom de rebolar no mato."]
-	}
-]
```
Comment (mvfsillva) [inline dialects/cearences.json]: First, thanks for contributing, your contribution is really very important for this project :]

---

Why was there a replace here?

Comment (Arieviloo) [reply dialects/cearences.json]: it was not meant to be a replacement but an addition

Comment (mvfsillva) [reply dialects/cearences.json]: Could you adjust please?

Comment (Arieviloo) [reply dialects/cearences.json]: I believe it was my vscode settings, it may have changed in time to save and left in a format that I use

**Comment by mvfsillva** [comment]
@Arieviloo do you still intend to solve the problem?

**Pull request closed without merging**